src/Files.App/Views/Properties/SecurityAdvancedPage.xaml

```diff
@@ -639,12 +687,12 @@
 				<Button
 					x:Name="ReplaceChildPermissionsButton"
 					Grid.Column="1"
-					x:Load="{x:Bind SecurityAdvancedViewModel.IsFolder, Mode=OneWay}"
+					x:Load="{x:Bind SecurityAdvancedViewModel.AccessControlList.IsFolder, Mode=OneWay}"
 					Command="{x:Bind SecurityAdvancedViewModel.ReplaceChildPermissionsCommand, Mode=OneWay}"
 					ToolTipService.ToolTip="{helpers:ResourceString Name=SecurityAdvancedReplaceChildPermissions2/ToolTipService/ToolTip}"
 					Visibility="Collapsed">
 					<TextBlock Text="{helpers:ResourceString Name=SecurityAdvancedReplaceChildPermissions/Text}" TextTrimming="CharacterEllipsis" />
-				</Button>
+				</Button>-->
```

Comment:
A reason to keep the code commented from l.655 to l.695?

Reply:
Disabled for now.